server_addon/max/client/ayon_max/plugins/load/load_pointcache.py

```diff
@@ -31,45 +31,50 @@ def load(self, context, name=None, namespace=None, data=None):
         file_path = self.filepath_from_context(context)
         file_path = os.path.normpath(file_path)
 
-        abc_before = {
-            c
-            for c in rt.rootNode.Children
-            if rt.classOf(c) == rt.AlembicContainer
-        }
-
-        rt.AlembicImport.ImportToRoot = False
-        rt.importFile(file_path, rt.name("noPrompt"), using=rt.AlembicImport)
-
-        abc_after = {
-            c
-            for c in rt.rootNode.Children
-            if rt.classOf(c) == rt.AlembicContainer
-        }
-
-        # This should yield new AlembicContainer node
-        abc_containers = abc_after.difference(abc_before)
-
-        if len(abc_containers) != 1:
-            self.log.error("Something failed when loading.")
-
-        abc_container = abc_containers.pop()
-        selections = rt.GetCurrentSelection()
-        for abc in selections:
-            for cam_shape in abc.Children:
-                cam_shape.playbackType = 0
-
-        namespace = unique_namespace(
-            name + "_",
-            suffix="_",
-        )
         abc_objects = []
-        for abc_object in abc_container.Children:
-            abc_object.name = f"{namespace}:{abc_object.name}"
-            abc_objects.append(abc_object)
-        # rename the abc container with namespace
-        abc_container_name = f"{namespace}:{name}"
-        abc_container.name = abc_container_name
-        abc_objects.append(abc_container)
+        product_fps = float(context["version"]["attrib"].get("fps"))
+        if product_fps is None:
+            # Just stick to current scene FPS
+            product_fps = float(rt.frameRate)
+        with scene_fps(product_fps):
```

Comment:
Preferably with this fix you wrap only the "smallest" amount of code that's needed for this fix.

In this case I'd expect you'd only need to do:

```python
with scene_fps(product_fps):
    rt.importFile(file_path, 
                         rt.name("noPrompt"), 
                         using=rt.AlembicImport)
```